resync/api/agent_evolution_api.py

```
@@ -548,26 +554,26 @@ async def get_performance_metrics(
+        if not AGENT_IMPROVEMENTS_DIR.exists():
+             AGENT_IMPROVEMENTS_DIR.mkdir(parents=True, exist_ok=True)
+             
         async with aiofiles.open(file_path) as f:
-            data = json.load(f)
+            data = json.loads(await f.read())
             return ImprovementSuggestion(**data)
     except Exception:
         return None
 
 
 async def _save_suggestion(suggestion: ImprovementSuggestion):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_save_suggestion`

Add return type annotation: `None`

(ANN202)